enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js

```diff
@@ -1,4 +1,51 @@
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
-  gitRawCommitsOpts: { grep: "(sdk)" },
+  gitRawCommitsOpts: {
+    grep: "(sdk)",
+  },
+  options: {
+    preset: "angular",
+    pkg: {
+      path: "enterprise/frontend/src/embedding-sdk/package.template.json",
+    },
+    tagPrefix: "embedding-sdk-",
+    transform: function (commit, cb) {
+      if (typeof commit.gitTags === "string") {
+        const tagPrefix = "embedding-sdk-";
+        const rtag = new RegExp(`tag:\\s*[=]?${tagPrefix}(.+?)[,)]`, "gi");
+        const match = rtag.exec(commit.gitTags);
+        rtag.lastIndex = 0;
+        if (match) {
+          commit.version = match[1];
+        }
+      }
+      if (commit.committerDate) {
+        const dateFormatter = Intl.DateTimeFormat("sv-SE", {
+          timeZone: "UTC",
+        });
+        commit.committerDate = dateFormatter.format(
+          new Date(commit.committerDate),
+        );
+      }
+
+      if (hasBackportPrefix(commit)) {
```

Comment:
This is the new code. This block made sure we cleanup the commit message that will be included in the changelog.

Since the original commit message doesn't follow conventional changelog, it doesn't have `type`, `scope`, or `subject` so I extract them here manually.